portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/databasepartitioning/DatabasePartitioning.testcase

```diff
@@ -565,6 +565,75 @@ definition {
 		}
 	}
 
+	@priority = 5
+	test PortalInstanceCreationDeletionUpdatesDBPartitionUtilWithCluster {
+		property app.server.bundles.size = "1";
+		property cluster.enabled = "true";
```

Comment:
@vicnate5 what happened to the "test cluster in integration test" thing?

We need shift focus to that, I don't want any more cluster tests done in this way.

Reply:
We have the same problems with upgrades, we whould probably find a complete solution for both use cases